shell.nix

```diff
@@ -1,19 +1,43 @@
 scope@{ pkgs ? import <nixpkgs> { } }:
+with pkgs;
+let
+  tokio-console = import ./default.nix { inherit pkgs; };
 
-let env = (import ./default.nix scope);
+  haveGlibcLocales = pkgs.glibcLocales != null && stdenv.hostPlatform.libc == "glibc";
+  glibcLocales-utf8 = (glibcLocales.override { locales = [ "en_US.UTF-8" ]; });
 
-in with pkgs;
+  env = buildEnv {
+    name = "console-env";
+    paths = [
+      direnv
+      binutils
+      stdenv
+      bashInteractive
+      docker
+      cacert
+      gcc
+      cmake
+      pkg-config
+      openssl
+    ]
+    ++ lib.optional haveGlibcLocales glibcLocales-utf8
+    ++ lib.optionals stdenv.isDarwin [ libiconv ]
```

Comment:
I removed a `Security` package from this list of optional deps. `nix flake check` couldn't find it, and I couldn't find a reference to it anywhere in `nixpkgs`. If it's from an overlay somewhere, I can add that as a flake input and bring the package back.